src/pages/surveys/[id]/questions.module.css

```diff
@@ -3,6 +3,7 @@
   cursor-pointer hover:bg-gray-400 transition-all duration-200;
 }
 
-.nextQuestionLink:disabled {
-  @apply opacity-50 cursor-not-allowed hover:bg-white;
+.leaveSurveyButton {
+  @apply w-7 h-7 absolute top-8 right-8 z-10 flex justify-center items-center bg-gray-400 rounded-full
+  cursor-pointer hover:bg-gray-600 transition-all duration-200;
```

Comment:
```suggestion
    cursor-pointer hover:bg-gray-600 transition-all duration-200;
```
Same indentation missing here?

Reply:
Updated in [e38244a](https://github.com/hanam1ni/react-surveyor/pull/93/commits/e38244a4df736e21c5156fc7a53850187611f59d)